swifter/swifter.py

```diff
@@ -1,13 +1,17 @@
+import timeit
+import warnings
 import pandas as pd
+
 from psutil import cpu_count
+from tqdm import tqdm
 from dask import dataframe as dd
-import timeit
-import warnings
+
 from tqdm import tqdm
-from .tqdm_dask_progressbar import TQDMDaskProgressBar
+from tqdm_dask_progressbar import TQDMDaskProgressBar
```

Comment:
Please return this to original state. It should be as follows:
`from .tqdm_dask_progressbar import TQDMDaskProgressBar`

The . is necessary for relative package imports when deploying on pip